ee/modules/600-flant-integration/docs/README_RU.md

```diff
@@ -103,3 +93,99 @@ kubectl annotate node test pricing.flant.com/nodeType=Special
+
+На основании собранных данных:
+- Рассчитывается стоимость услуги [Managed Kubernetes](https://flant.ru/services/managed-kubernetes-as-a-service).
+- Происходит планирование миграций.
+- Контролируется гомогенность инфраструктуры.
+
+Среднее количество sample’ов отсылаемых одним кластером: **35 строк каждые 30 секунд**.
+
+Пример собранных данных:
+![](../../images/600-flant-integration/image1.png)
+![](../../images/600-flant-integration/image6.png)
+
+Дополнительно к метрикам собираемым модулем flant-integration, с помощью модуля [upmeter](../500-upmeter/) производится сбор [метрик доступности](#метрики-sla) для анализа выполнения SLA.
+
+#### Оповещения в Madison
+
+Madison — сервис обработки оповещений в составе платформы мониторинга компании Флант. Madison может принимать уведомления в формате Prometheus. 
+
+При создании нового кластера Deckhouse:
+1. При помощи ключа лицензии происходит автоматическая регистрация кластера в Madison.
+2. Madison выдаёт кластеру ключ, необходимый для отправки уведомлений.
+3. При помощи DNS-запроса для домена `madison-direct.flant.com` Deckhouse находит все доступные на текущий момент IP-адреса Madison.
+4. Для каждого адреса создается Pod `madison-proxy`, в который Prometheus отправляет уведомления.
+
+Схема отправки оповещений из кластера в Madison:
+![](../../images/600-flant-integration/image5.png)
+
+В среднем скорость отправки оповещений из кластера равняется **2kb/s**. Но здесь следует учитывать, что чем больше инцидентов произошло в кластере, тем больше данных будет отправлено.
+
+#### Логи оператора Deckhouse
+Оператор Deckhouse является центральным компонентом всего кластера. Для того чтобы иметь данные для отладки во всех обслуживаемых кластерах, в операторе Deckhouse организована отправка логов при помощи модуля [log-shipper](../460-log-shipper/) в хранилище Loki компании Флант (не напрямую, а также через сервис Connect).
+
+В логах **нет секретных данных** касающихся кластера (примеры сообщений приведены на рисунке ниже). Собранная информация помогает понять, какие действия выполнял оператор и когда он их выполнял.
+
+Пример логов оператора Deckhouse:
+![](../../images/600-flant-integration/image4.png)
+
+При смене релиза Deckhouse, количество отправляемых данных логов достигает в среднем **150 записей в минуту**. При обычной работе, количество отправляемых данных логов достигает в среднем **20 записей в минуту**.
+
+#### Метрики SLA
+
+С помощью модуля [upmeter](../500-upmeter/) в кластере выполняется измерение уровня доступности различных компонентов. Такие метрики необходимы для определения уровня доступности компонентов платформы и анализа выполнения соглашения об уровне сервиса (SLA, Service Level Agreement).  
```

Comment:
This section refers to the upmeter module.  Maybe it's better to move this block to the upmeter module and link it here?
@nabokihms

Reply:
I agree. I think it deserves at most a couple of sentences.

> Upmeter sends availablility metrics, that lets Flant control SLA per cluster. The metrics throughput is about **100 samples sent every 5 minutes**, the number of samples being dependent on the set of enabled modules in Deckhouse.

After that we can give a link to dedicated Upmeter page.

Reply:
Thanks. Refactored.